### PR TITLE
Mlutils 1

### PR DESCRIPTION
**Pull request by yjb-ds**

### CODE REVIEW

File: mlrun/mlutils/__init__.py [hunk 1]
```diff
@@ -4,8 +4,17 @@
                      gen_sklearn_model,
                      eval_class_model)
 
-from .plots import (gcf_clear, plot_roc,
-                    plot_importance)
```
Comment (yaronha) [inline mlrun/mlutils/__init__.py:8]: @yjb-ds we may need to keep `plot_roc` for backwards compatibility

File: mlrun/mlutils/plots.py [hunk 2]
```diff
@@ -16,99 +17,235 @@ def gcf_clear(plt):
...
 
-    fname = f"{plots_dest}/{key}.html"
-    context.log_artifact(PlotArtifact(key, body=plt.gcf()), local_path=fname)
-
-    # feature importances are also saved as a csv table (generally small):
-    fname = key + "-tbl.csv"
-    return context.log_artifact(TableArtifact(key + "-tbl", df=feature_imp), local_path=fname)
-
-
-def plot_roc(
```
Comment (yaronha) [inline mlrun/mlutils/plots.py:57]: @yjb-ds we should keep it for now so we wont break existing funcs

File: docker-build.sh [hunk 3]
```diff
@@ -26,9 +27,24 @@ do
     docker push $REPO/$PREFIX-$IMAGE:$NEW_TAG
 done
 
-for IMAGE in 'dask' 'mlrun-api' 'test'
+# need some python 3.6 for (legacy) package consistency 
+for IMAGE in 'base' 'models' 'models-gpu'
```
Comment (yaronha) [inline docker-build.sh:31]: @yjb-ds would be nice to have a partial build option (only some containers or split to 2-3 groups), it usually kills me VM disk/mem space to build all, or sometimes i need just 1-2 images and need to wait 40min for all the build

File: docker-build.sh [hunk 4]
```diff
@@ -26,9 +27,24 @@ do
...
+        --build-arg REPO=$REPO \
+        --build-arg PREFIX=$PREFIX \
+        --build-arg NEW_TAG=$NEW_TAG \
+        --build-arg PYTHON_VER=$PYTHON_VER_CORE \
+        -t $REPO/$PREFIX-$IMAGE:$NEW_TAG-py"${PYTHON_VER_CORE//.}" .
+
+    docker push $REPO/$PREFIX-$IMAGE:$NEW_TAG-py"${PYTHON_VER_CORE//.}"
+done
+
+for IMAGE in 'dask' 'mlrun-api' 'test'
```
[the remark this answers is not in the document]
Comment (yjb-ds) [reply docker-build.sh:45]: no we don't

File: docker-build.sh (hunk 3 above)
Comment (yjb-ds) [reply docker-build.sh:31]: this will be dealt with in next update

**Pull request merged**